### DISCUSSION TITLE
Bulk total RNA vs mRNA paired end sequencing data

### DISCUSSION THREAD

**Discussion by christoforos-dimitropoulos** [milaboratory/mixcr#1798] [Q&A] [answered]
Hello,

I have some bulk sequencing data of sorted mouse CD4 T cells which I would like to import to mixcr. I have some fastq files (R1 and R2 for each sample,100nt each) and some samples were mRNA enriched before sequencing while others were not (total RNA was sequenced).
At the moment I am using:
mixcr analyze rna-seq \
    --species mmu \
    sample1_R1_001.fastq.gz \
	sample1_R2_001.fastq.gz \
    results/CD4
 and I would like to acquire the TCR clones for each sample. Is this the correct way to do it? Any advice or tip would be appreciated :) Thanks a lot!

**Comment by mizraelson** [comment, marked as answer]
If this is a regular fragmented RNA-seq data, then the command is correct.
